tutoriaux/CNN/CNN_solutions.ipynb

```diff
@@ -1244,14 +1244,14 @@
     "        super(LeNet5, self).__init__()\n",
     "        self.block1 = nn.Sequential(\n",
     "            nn.Conv2d(1, 16, kernel_size=5, padding=2),\n",
-    "            nn.BatchNorm2d(16),\n",
     "            nn.ReLU(),\n",
+    "            nn.BatchNorm2d(16),\n",
```

Comment:
According, to pytorch [source code](https://github.com/pytorch/vision/blob/master/torchvision/models/vgg.py#L71) and the [original paper](https://arxiv.org/pdf/1502.03167.pdf), batch norm goes before the activation. Why do you think it's a good idea to invert that?

Reply:
I remember a paper where they tried the different combinations and Relu -> BN did better. Looking for it now, I can only find [this](https://github.com/keras-team/keras/issues/1802#issuecomment-187966878). I'm not sure what the current consensus is now. It doesn't matter at all for this task though, we can leave BN -> Relu.

Reply:
Sure, but as it is an intro tutorial I would stick to the original paper.